WebAPI.Tests/WebAPITest.cs

```diff
@@ -57,30 +57,30 @@ public void POST_With_Everything_Good()
             Book book = new Book() { id =15,price=140,title="Book twelve",author="Neelesh ji",category="Fiction"};
             Response response = bookservice.Post(book);
             List<Book> booklist = response.booklist;
-            string msg = response.message;
+            string responseMessage = response.message;
 
-            Assert.Equal("Data Added Successfully !", msg);
+            Assert.Equal("Data Added Successfully !", responseMessage);
             response = bookservice.Delete(15);
         }
 
         [Fact]
-        public void POST_Input_with_negetive_Id_and_price()
+        public void POST_Input_with_negetive_Book_Id_and_price()
         {
             Book book = new Book() { id = -1, price = -140, title = "Book twelve", author = "Neelesh ji", category = "Fiction" };
             Response response = bookservice.Post(book);
             List<Book> booklist = response.booklist;
-            string msg = response.message;
-            Assert.Equal("Error: Invalid Book ID or Price, they should be a positive number!", msg);
+            string responseMessage = response.message;
+            Assert.Equal("Error: Invalid Book ID or Price, they should be a positive number!", responseMessage);
```

Comment:
We can consider using FluentAssertions in these test cases instead Assert.Equal, as FluentAssertions allow us to compare objects instead of just comparing strings of error messages...